src/gui/owncloudgui.cpp

```diff
@@ -544,12 +538,24 @@ void ownCloudGui::slotShowGuiMessage(const QString &title, const QString &messag
 
 void ownCloudGui::slotShowSettings()
 {
-    if (_settingsDialog.isNull()) {
-        _settingsDialog = new SettingsDialog(this);
-        _settingsDialog->setAttribute(Qt::WA_DeleteOnClose, true);
-        _settingsDialog->show();
+    // to raise the dialog first
+    slotShowMainDialog();
+    _settingsDialog->showSettingsPage();
+}
+
+void ownCloudGui::slotShowMainDialog()
```

Comment:
This is wrongly named, this really shows the settings dialog not the main dialog.

Reply:
Yup.